scalableminds/webknossos
# Add description to explorative annotations view

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to the explorative annotations view in the dashboard
- hover over the names of each explorative annotation 
- the descriptions should show up
- if no description exists, there should be a default text

### Issues:
- fixes [#2922](https://github.com/scalableminds/webknossos/issues/2922)

------
